core/contract/src/test/java/org/eclipse/dataspaceconnector/contract/negotiation/ContractNegotiationIntegrationTest.java

```diff
@@ -23,29 +25,36 @@
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 
-import java.util.concurrent.TimeUnit;
+import java.time.Duration;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
 import static org.eclipse.dataspaceconnector.spi.types.domain.contract.negotiation.ContractNegotiationStates.CONFIRMED;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 @ComponentTest
 class ContractNegotiationIntegrationTest extends AbstractContractNegotiationIntegrationTest {
 
+    private static final Duration DEFAULT_TEST_TIMEOUT = Duration.ofSeconds(15);
+    private static final Duration DEFAULT_POLL_INTERVAL = Duration.ofSeconds(1);
```

Comment:
nit: no need for such a "long" interval I think, they add up between tests, so something like 0.1s should do